### ISSUE TITLE
Accessibility: Form elements do not have associated labels

### ISSUE THREAD

**Issue by irina-baeva** [uiwjs/react-color#165]
I am encountering an accessibility issue when using the <Sketch> component from your library. The violation reported is:
```
Form elements do not have associated labels
```

That's also could be reproducible if you run Lighthouse report https://uiwjs.github.io/react-color/
